chapters/equations.tex

```diff
@@ -771,14 +771,14 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \lstinline!start!-values of variables having \lstinline!fixed = false! can be used as initial guesses, in case iterative solvers are used in the initialization phase.
 
 \begin{nonnormative}
-In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value of the \lstinline!start!-attribute as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
+In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
 \end{nonnormative}
 
-If a parameter has a modifier for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the modifier for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
+If a parameter has a value for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the value for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
```

Comment:
Maybe we could simplify our language further?
```suggestion
If a parameter has a \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that value.
```

Reply:
Yes, and it is in the light of this that I think we should take the chance to find an easier way to express ourselves; everyone knows that every `Real` variable has all these attributes, so it feels safe to say _has an X-attribute_ when meaning that the attribute has a value.  I think this would be one of the real benefits of this PR, allowing us to take a long step away from the complicated _has a modifier for the X-attribute_.

Reply:
By the way, how to word a thing like this seems like a perfect question to ask the language group.

Reply:
Ok, but preferably separate from this PR; as it will likely impact many other texts.

Reply:
OK, I'll create a separate issue.  Resolving.